docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-imageless-card-group/__tests__/cgdp-imageless-card-group.test.ts

```diff
@@ -0,0 +1,218 @@
+
+		// Create the JS
+		cgdpImagelessCardInit();
+
+		const card = screen.getAllByRole('link');
+
+		fireEvent.click(card[0]);
+
+		expect(trackOtherSpy).toHaveBeenCalledWith(
+			'MLP:ImagelessCard:LinkClick',
+			'MLP:ImagelessCard:LinkClick',
+			{
+				location: 'Body',
+				pageType: 'cgvMiniLanding',
+				pageTemplate: 'ncids_default',
+				pageRowVariant: 'Not Defined',
+				pageRows: 1,
+				pageRowIndex: 1,
+				pageRowCols: 0,
+				pageRowColIndex: 0,
+				containerItems: 3,
+				containerItemIndex: 1,
+				componentType: 'Imageless Card',
+				componentTheme: 'Not Defined',
+				componentVariant: 'Not Defined',
+				title: 'The Optional Imageless Card Group Heading - two Ca',
+				linkType: 'External',
+				linkText: 'Not Defined',
+				linkArea: 'Not Defined',
+				totalLinks: 1,
+				linkPosition: 1,
+			}
+		);
+	});
+
+	it('internal imageless card', () => {
+		const dom = cgdpImagelessCardRowDom();
+
+		// Lets make a spy to ensure that trackOther is called correctly
+		const trackOtherSpy = jest.spyOn(eddlUtil, 'trackOther');
+
+		// Inject the HTML into the dom.
+		document.body.insertAdjacentHTML('beforeend', dom.outerHTML);
+
+		// Create the JS
+		cgdpImagelessCardInit();
+
+		const card = screen.getAllByRole('link');
+
+		fireEvent.click(card[1]);
+
+		expect(trackOtherSpy).toHaveBeenCalledWith(
+			'MLP:ImagelessCard:LinkClick',
+			'MLP:ImagelessCard:LinkClick',
+			{
+				location: 'Body',
+				pageType: 'cgvMiniLanding',
+				pageTemplate: 'ncids_default',
+				pageRowVariant: 'Not Defined',
+				pageRows: 1,
+				pageRowIndex: 1,
+				pageRowCols: 0,
+				pageRowColIndex: 0,
+				containerItems: 3,
+				containerItemIndex: 2,
+				componentType: 'Imageless Card',
+				componentTheme: 'Not Defined',
+				componentVariant: 'Not Defined',
+				title: 'The Optional Imageless Card Group Heading - two Ca',
+				linkType: 'Internal',
+				linkText: 'Not Defined',
+				linkArea: 'Not Defined',
+				totalLinks: 1,
+				linkPosition: 1,
+			}
+		);
+	});
+
+	it('multimedia imageless card', () => {
+		const dom = cgdpImagelessCardRowDom();
+
+		// Lets make a spy to ensure that trackOther is called correctly
+		const trackOtherSpy = jest.spyOn(eddlUtil, 'trackOther');
+
+		// Inject the HTML into the dom.
+		document.body.insertAdjacentHTML('beforeend', dom.outerHTML);
+
+		// Create the JS
+		cgdpImagelessCardInit();
+
+		const card = screen.getAllByRole('link');
+
+		fireEvent.click(card[2]);
+
+		expect(trackOtherSpy).toHaveBeenCalledWith(
+			'MLP:ImagelessCard:LinkClick',
+			'MLP:ImagelessCard:LinkClick',
+			{
+				location: 'Body',
+				pageType: 'cgvMiniLanding',
+				pageTemplate: 'ncids_default',
+				pageRowVariant: 'Not Defined',
+				pageRows: 1,
+				pageRowIndex: 1,
+				pageRowCols: 0,
+				pageRowColIndex: 0,
+				containerItems: 3,
+				containerItemIndex: 3,
+				componentType: 'Imageless Card',
+				componentTheme: 'Not Defined',
+				componentVariant: 'Not Defined',
```

Comment:
this should be set as 1,2 or 3 card row to reflect actual behavior

Reply:
same for internal and external events

Reply:
@KateMashkinaNIH if this was done can you resolve this conversation please?